server.js

```diff
@@ -47,10 +47,7 @@ if ('production' === process.env.NODE_ENV || 'test' === process.env.NODE_ENV) {
       }
 
       const document = JSON.parse(data);
-
-      if (document.readonly) {
-        delete(document.readonly);
-      }
+      document.readonly = !!document.readonly;
```

Comment:
It adds the boolean attribute `readonly` to the document, in case the attribute does not exist.

`!!var.foo` returns `true` if the var contains a property `foo` that is evaluated to true.